features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/model/EnabledFeature.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.features.preferences.impl.model
+
+import io.element.android.libraries.featureflag.api.Feature
+
+data class EnabledFeature(
```

Comment:
`EnableableFeature` would be a better name, but the word `enableable` does not seem to exist. Suggestion welcome!